Library/Homebrew/utils.rb

```diff
@@ -291,6 +291,16 @@ def paths
   end.uniq.compact
 end
 
+# return the shell profile file based on users' preference shell
+def shell_profile
+  case ENV["SHELL"]
+  when %r{/(ba)?sh} then "~/.bash_profile"
+  when %r{/zsh} then "~/.zshrc"
+  when %r{/ksh} then "~/.kshrc"
+  else "~/.profile"
```

Comment:
Oh, I just find out I misunderstood your previous comment. I'm cool to default to `.bash_profile`.